README.md

```diff
@@ -17,11 +17,16 @@ Download your preferred asset from the [releases page](https://github.com/helm/c
 
 ```console
 $ # clone repo to some directory outside GOPATH
-$ git clone github.com/helm/chart-releaser
+$ git clone https://github.com/helm/chart-releaser
+$ cd chart-releaser
 $ go mod download
-$ go install
+$ go install ./...
 ```
 
+### Docker (for Continuous Integration)
+
+Docker images are pushed to the quay.io/helmpack/chart-releaser Docker repository. The Docker image is built on top of Alpine and its default entry-point is cr. See the `Dockerfile` for more informations about the Docker image.
```

Comment:
Some suggestions:

```suggestion
Docker images are pushed to the [helmpack/chart-releaser](https://quay.io/repository/helmpack/chart-releaser?tab=tags) Quay container registry. The Docker image is built on top of Alpine and it's default entry-point is `cr`. See the [Dockerfile](./Dockerfile) for more information about the Docker image.
```

Reply:
Done. Is there a squashing mechanism in this repo or should I manually squash?